AmahiAnywhere/AmahiAnywhere/Presentation/Login/LoginViewController.swift

```diff
@@ -29,6 +29,11 @@ class LoginViewController: BaseUIViewController {
         (textField as! SkyFloatingLabelTextField).errorMessage = nil
         return true
     }
+    
+    @IBAction func userClickForgotPassword(_ sender: UIButton) {
+        UIApplication.shared.open(NSURL(string:"https://www.amahi.org/forgot")! as URL)
+    }
+    
     @IBAction func userclickSignIn(_ sender: Any) {
         
         if (usernameInputField.text?.isEmpty)! {
```

Comment:
force unwrapping while defining is not good (it can crash the app), you should use `if let` or `guard let` statements instead.

You could possibly have done:
 `if let text = usernameInputField.text, text.isEmpty {`